js/actions.js

```diff
@@ -1,7 +1,29 @@
+// Selectors for search button
+//TODO - change id names HTML dependant
+var searchButton = $("#searchBtn");
+var searchInput = $("#searchValue");
 
-// !!! remove before writing actual code !!!
-console.log(`
-||||||||||||||||||||||||||||||||||||||||||||||||
-| It's not a bug; it's an undocumented feature |
-||||||||||||||||||||||||||||||||||||||||||||||||`
-)
\ No newline at end of file
+//On click function for search button
+searchButton.on("click", function(event){
+    event.preventDefault();
+    var userInput = searchInput.val().trim();
+//Tasty API Query one - list of meals
+    const settings = {
+        "async": true,
+        "crossDomain": true,
+        "url": "https://tasty.p.rapidapi.com/recipes/list?from=0&size=8&q=Chicken",
```

Comment:
Here the query is always going to search for "Chicken".  You probably want to feed it the variable "userInput" instead.

Reply:
Also, "size=8" is probably not what we want to search, but rather "size=1" and then limit the result of its "results" to 8.

Reply:
I believe size is the number of results (see screenshot below)
![image](https://user-images.githubusercontent.com/116746547/216376728-3e1efec7-b5cb-422b-a08a-fc83ebb36a0d.png)

Reply:
It is, but if you go into each element of this response, you will notice number 2-8 aren't actually results that are useful to us.

Reply:
This is the API that just pulls the ID number, not the API that gives us recipie information, so only the ID number is a valid output, which is then fed into the second tasty API, see below:
![image](https://user-images.githubusercontent.com/116746547/216378601-554d01d2-749e-4a4c-9855-ab1ea17a7b21.png)

The second image below is one of those id's put into the second API
![image](https://user-images.githubusercontent.com/116746547/216378811-7af66b22-38d0-4639-915e-ef4128fa15b8.png)